### PR TITLE
feat: add issue badge

### LINKED ISSUE #53: [Docs] Add shield/ badges for contributors and issues count
**Current State:**

There are no shields/badges showing contributor and issue count

**Desired State:**

Add contributor and issues count shield/ badges to readme

This is what a sample badge/shield looks like  
image

They can be generated for our repo from  [shields.io](https://shields.io)

### PR DESCRIPTION
**Pull request by schmelto**
resolve #53

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by sudhay23** [comment]
@schmelto Can you put up a screenshot of the result?

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # ShopOnline - WebApp (Celebrating open-source 🎉)
 
+[![open issues](https://img.shields.io/github/issues/sudhay23/ShopOnline-Web-App)](https://github.com/sudhay23/ShopOnline-Web-App/issues)
```
Comment (sudhay23) [inline README.md:3]: Screenshot can be put up

**Review by sudhay23 — CHANGES_REQUESTED** [review]

**Comment by san-coding** [comment]
@schmelto please resolve the merge conflict by keeping patch 1 and deleting the main (delete lines 3, 7, 8, 9)
image

**Comment by san-coding** [comment]
> @schmelto Can you put up a screenshot of the result?

@sudhay23 you can click on the file options and select view file to view readme changes

**Comment by sudhay23** [comment]
> > @schmelto Can you put up a screenshot of the result?
> 
> @sudhay23 you can click on the file options and select view file to view readme changes

Oops yeah. Cool then. Looks good.

**Review by sudhay23 — APPROVED** [review]

**Review by san-coding — CHANGES_REQUESTED** [review]
add contributor count badge and resolve merge conflicts by keeping patch 1 and deleting the main (delete lines 3, 7, 8, 9)
image

**Comment by schmelto** [comment]
image

**Review by san-coding — APPROVED** [review]
Thank you,  we appreciate your effort and contributions 🎉

**Pull request merged**